src/lib/state.ts

```diff
@@ -422,8 +420,6 @@ export const dumpConfigurationMetadata = (
     unit: metadata.unit,
     valueSize: metadata.valueSize,
     format: metadata.format,
-    name: metadata.name,
-    info: metadata.info,
```

Comment:
These properties were never really working. Does this removal need versioning?

Reply:
we don't know whether consuming applications are expecting this data int he payload for e.g. validation, so safest to keep it in